telegram.ext.dispatcher - ERROR - An uncaught error was raised while processing the update
Traceback (most recent call last):
  File "/Users/shirokov_sp/LearnPython/lesson3/env/lib/python3.6/site-packages/telegram/ext/dispatcher.py", line 279, in process_update
    handler.handle_update(update, self)
  File "/Users/shirokov_sp/LearnPython/lesson3/env/lib/python3.6/site-packages/telegram/ext/commandhandler.py", line 173, in handle_update
    return self.callback(dispatcher.bot, update, **optional_args)
  File "bot.py", line 33, in planet1
    text = message.text
NameError: name 'message' is not defined
telegram.ext.updater - INFO - Received signal 2 (SIGINT), stopping...
